common/components/WalletDecrypt/components/Trezor.tsx

```diff
@@ -133,7 +145,16 @@ export class TrezorDecrypt extends Component<Props, State> {
     this.setState({
       publicKey: '',
       chainCode: '',
-      dPath: DEFAULT_PATH
+      dPath: this.props.dPath.value
     });
   }
 }
+
+function mapStateToProps(state: AppState): StateProps {
+  const network = getNetworkConfig(state);
+  return {
+    dPath: getSingleDPath(SecureWalletName.TREZOR, network)
+  };
+}
+
+export const TrezorDecrypt = connect(mapStateToProps)(TrezorDecryptClass);
```

Comment:
To reduce # of connected components (Which we should strive to do more for performance, imo), I think we should just send each wallet the `network` and they can run this in constructor or whatever. It needs it anyway to disable certain wallets on certain networks.

Reply:
Agreed

Reply:
Addressed in https://github.com/MyEtherWallet/MyEtherWallet/pull/849/commits/43b5833c54d5a87abc686501b8478014c8455e86